SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -4326,7 +4337,15 @@ class CSolver {
    * \return Struct holding 4 su2doubles.
    */
   virtual StreamwisePeriodicValues GetStreamwisePeriodicValues() const { return StreamwisePeriodicValues(); }
-
+  
+  /*!
+   * \brief Save snapshot or POD data using libROM
+   * \param[in] solver - Container vector with all the solutions.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] config - Definition of the particular problem.
+   * \param[in] converged - Whether or not solution has converged.
+  */
+  void SavelibROM(CSolver** solver, CGeometry *geometry, CConfig *config, bool converged);
```

Comment:
```suggestion
   * \param[in] geometry - Geometrical definition of the problem.
   * \param[in] config - Definition of the particular problem.
   * \param[in] converged - Whether or not solution has converged.
  */
  void SavelibROM(const CGeometry *geometry, const CConfig *config, bool converged);
```
It looks like you don't need the solver container in that function? Or are you planning to add turbulence variables at some point?

Reply:
Good catch - it is not needed